estuary/utils/general.py

```diff
@@ -277,68 +277,103 @@ def story_flow(label):
+            'forward_relationship_display': 'triggered by',
             'forward_label': FreshmakerEvent.__label__,
+            'forward_label_display': 'Freshmaker event',
             'backward_relationship': '{0}>'.format(
                 Advisory.attached_builds.definition['relation_type']),
-            'backward_label': KojiBuild.__label__
+            'backward_relationship_display': 'attached to',
+            'backward_label': KojiBuild.__label__,
+            'backward_label_display': 'Koji build'
         }
     elif label == FreshmakerEvent.__label__:
         return {
             'uid_name': FreshmakerEvent.id_.db_property or FreshmakerEvent.id.name,
+            'display_label': 'Freshmaker event',
             'forward_relationship': '{0}>'.format(
                 FreshmakerEvent.triggered_container_builds.definition['relation_type']),
+            'forward_relationship_display': 'triggered by',
             'forward_label': ContainerKojiBuild.__label__,
+            'forward_label_display': 'Container Koji builds',
             'backward_relationship': '{0}>'.format(FreshmakerEvent.triggered_by_advisory
                                                    .definition['relation_type']),
-            'backward_label': Advisory.__label__
+            'backward_relationship_display': 'triggered by',
+            'backward_label': Advisory.__label__,
+            'backward_label_display': 'Advisory'
         }
     elif label == ContainerKojiBuild.__label__:
         return {
             'uid_name': ContainerKojiBuild.id_.db_property or ContainerKojiBuild.id.name,
+            'display_label': 'Container Koji build',
             'forward_relationship': '{0}<'.format(
                 ContainerKojiBuild.advisories.definition['relation_type']),
+            'forward_relationship_display': 'attached to',
             'forward_label': ContainerAdvisory.__label__,
+            'forward_label_display': 'Container advisory',
             'backward_relationship': '{0}<'.format(
                 ContainerKojiBuild.triggered_by_freshmaker_event.definition['relation_type']),
-            'backward_label': FreshmakerEvent.__label__
+            'backward_relationship_display': 'triggered by',
+            'backward_label': FreshmakerEvent.__label__,
+            'backward_label_display': 'Freshmaker event'
         }
     elif label == ContainerAdvisory.__label__:
         return {
             'uid_name': ContainerAdvisory.id_.db_property or ContainerAdvisory.id.name,
+            'display_label': 'Container advisory',
             'forward_relationship': None,
+            'forward_relationship_display': None,
             'forward_label': None,
+            'forward_label_display': None,
             'backward_relationship': '{0}>'.format(
                 ContainerAdvisory.attached_builds.definition['relation_type']),
-            'backward_label': ContainerKojiBuild.__label__
+            'backward_relationship_display': 'attached to',
```

Comment:
The wording is wrong here because it is from the perspective of the ContainerAdvisory, not the ContainerKojiBuild

Reply:
how should the wording be here, I'm unsure @mprahl

Reply:
So for example if it's from a perspective of a ContainerKojiBuild, the wording is correct since 'Container Koji builds are attached to a Container Advisory' and not the other way around. So for ContainerAdvisory, you can have something like 'Container Advisory having attached Container Koji Builds'. I couldn't find the exact wordings for the second statement. But I hope this helps

Reply:
Either what @yashvardhannanavati suggested or you could make it something generic like `contains`.